packages/apps/package.json

```diff
@@ -11,20 +11,20 @@
   },
   "sideEffects": true,
   "type": "module",
-  "version": "0.98.2-133",
+  "version": "0.98.2-136",
   "dependencies": {
     "@babel/core": "^7.16.0",
     "@babel/polyfill": "^7.12.1",
     "@babel/runtime": "^7.16.3",
-    "@polkadot/apps-config": "^0.98.2-133",
-    "@polkadot/apps-routing": "^0.98.2-133",
+    "@polkadot/apps-config": "^0.98.2-136",
+    "@polkadot/apps-routing": "^0.98.2-136",
     "@polkadot/dev": "^0.63.37",
-    "@polkadot/react-api": "^0.98.2-133",
-    "@polkadot/react-components": "^0.98.2-133",
-    "@polkadot/react-hooks": "^0.98.2-133",
-    "@polkadot/react-params": "^0.98.2-133",
-    "@polkadot/react-query": "^0.98.2-133",
-    "@polkadot/react-signer": "^0.98.2-133",
+    "@polkadot/react-api": "^0.98.2-136",
```

Comment:
Not sure where these come from. Should not be there.

Reply:
Yes pretty strange, I tried to build it locally to be sure all is ok, is it it auto-incremented at this time maybe ?

Reply:
Should I redo a commit removing all the version changes ?

Reply:
Yes, please do :) Sorry about that, generally they are only incremented with the gh release build. This one throws it a bit out :)

Reply:
closing, pushed a new one :)